### PR TITLE
Example app

### PR DESCRIPTION
**Pull request by tpom6oh**

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  playButtonTitle() {
+    if (this, this.state.playing) {
+      return "Pause";
+    } else {
+      return "Play";
+    }
+  }
+
+  loopButtonTitle() {
+    if (this, this.state.looping) {
```
Comment (arnihermann) [inline example/App.js]: this looks like it's missing something

File: example/rn-cli.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+module.exports = {
+    resolver: {
+        blacklistRE: blacklist([
```
Comment (arnihermann) [inline example/rn-cli.config.js]: what's this?

Comment (tpom6oh) [reply example/rn-cli.config.js]: It's some config I used when trying to build with npm, not needed anymore

File: src/AvoDebugger.js [hunk 3]
```diff
@@ -72,23 +79,13 @@ export default class AvoDebugger extends Component {
...
-      eventProps: eventProperties,
-      userProps: userProperties
+  componentDidMount() {
+    this.setState(prevState => ({unreadMesages: AvoDebugger.unhandledNewItems.count}), () => {
+      AvoDebugger.unhandledNewItems.count = 0;
     });
+  }
+
+  onNewEvent() {
     this.setState(prevState => ({unreadMesages: prevState.unreadMesages + 1}));
```
Comment (arnihermann) [inline src/AvoDebugger.js]: didn't notice it before but there's a typo in the variable `unreadMesages`, should be `unreadMessages`

File: example/App.js (hunk 1 above)
Comment (arnihermann) [reply example/App.js]: should this be `if (this.state.looping)` ?

**Comment by arnihermann** [comment]
LGTM

**Pull request merged**